ykrt/build.rs

```diff
@@ -41,18 +39,17 @@ fn main() {
 
     // If we symlink libykshim.so into the target dir, then this is already in the linker path when
     // run under cargo. In other words, the user won't have to set LD_LIBRARY_PATH.
-    let mut sym_dest = out_dir.clone();
+    let mut sym_dest = PathBuf::from(env::var("OUT_DIR").unwrap());
     sym_dest.push("..");
     sym_dest.push("..");
     sym_dest.push("..");
     sym_dest.push(YKSHIM_SO);
     if !PathBuf::from(&sym_dest).exists() {
-        let mut sym_src = internal_dir.clone();
-        sym_src.push("target");
-        sym_src.push("release");
-        sym_src.push(YKSHIM_SO);
-        dbg!(&sym_src, &sym_dest);
-        symlink(sym_src, sym_dest).unwrap();
+        internal_dir.push("target");
```

Comment:
I think keeping `let mut sym_src = internal_dir;` would be nicer as after these pushes it is no longer contains the internal dir. Just remove the `.clone()`.

Reply:
Makes sense. Fixed in e134a8216b2188fb00ac6ed9f3f6545de7016bf6.